spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -200,11 +212,14 @@ const ConfigSteps = () => {
 		await OAuthManagerInstance.installNewApp(() => {
 			getOrganizations();
 		});
-	}
+	};
 
 	return (
 		<Wrapper>
 			<SyncHeader />
+			{
+				error && <Error type={error.type} message={error.message} />
```

Comment:
nit: could we please use ternary instead? I find this construction more difficult to parse... :coneofshame: